src/account.ts

```diff
@@ -370,13 +373,15 @@ export class Account {
 
     // wei -> tokens
     public async weiToHuman(amountWei: bigint): Promise<string> {
-        return this.getClient().fromBaseUnit(amountWei.toString());
+        const tokenAddress = this.config.pools[this.getCurrentPool()].tokenAddress;
+        return await this.getClient().fromBaseUnit(tokenAddress, amountWei.toString());
     }
 
 
     public async getBalance(): Promise<[string, string]> {
+        const tokenAddress = this.config.pools[this.getCurrentPool()].tokenAddress;
         const balance = await this.getClient().getBalance();
-        const readable = this.getClient().fromBaseUnit(balance);
+        const readable = await this.getClient().fromBaseUnit(tokenAddress, balance);
```

Comment:
It seems that we shouldn't use `tokenAddress` here. 
![image](https://github.com/zkBob/zkbob-console/assets/17165678/6f82e0dd-0141-48bd-a694-463c74c18cad)
![image](https://github.com/zkBob/zkbob-console/assets/17165678/4edf668c-ae6c-4c35-930e-192175860855)

Reply:
Oh thanks! Fixed in https://github.com/zkBob/zkbob-console/pull/85/commits/2cde248b397e35f50b8c03c52c4c5249e1f09c8f

Reply:
And also found another bugs with the same conversion https://github.com/zkBob/zkbob-console/pull/85/commits/6c4b78b47157a4706c40cd054120111e6d39388c